apps/src/applab/AppLabView.jsx

```diff
@@ -206,7 +206,7 @@ var AppLabView = React.createClass({
 module.exports = connect(function propsFromStore(state) {
   return {
     isReadOnlyWorkspace: state.level.isReadOnlyWorkspace,
-    instructionsInTopPane: state.level.instructionsInTopPane,
+    instructionsInTopPane: state.level.instructionsInTopPane && !!state.level.instructionsMarkdown,
```

Comment:
Hm... not sure if best to do this

* Here in `connect()`
* Inside the component, closer to where the actual exception would occur (since we are passing both properties into the component anyway)
* On initialize, before we ever push `instructionsInTopPane` into the redux store.

As the 'middle' choice, this feels a little weird to me, but I'm not sure I have a strong argument that one of the others is better.